NEW_CONNECTION.md

```diff
@@ -0,0 +1,105 @@
+# Add New Data Source
+
+## Images
+It is recommended to add the images as png and a good idea to include.  
```

Comment:
## Icons

The connector must provide an icon to be displayed on the selection tab.